.gitattributes

```diff
@@ -0,0 +1,7 @@
+* text=auto
```

Comment:
I wasn't familiar with `export-ignore`. Good stuff!

Reply:
Found it a while ago, yep, good stuff!